src/views/assets/Cloud/Strategy/components/const.js

```diff
@@ -6,6 +6,7 @@ export const resourceTypeOptions = [
   { label: i18n.t('Node'), value: 'node' },
   { label: i18n.t('Zone'), value: 'domain' },
   { label: i18n.t('AccountTemplate'), value: 'account_template' },
+  { label: i18n.t('Tags'), value: 'label' },
   { label: i18n.t('Strategy'), value: 'name_strategy' }
 ]
 
```

Comment:
The current version of your code appears to be well-written and free from any obvious errors or inconsistencies. To optimize this function with JavaScript/Nodejs, you might look at using an array instead of a separate variable in `resourceTypeOptions`. This would make it clear that each element represents something new within the array:

```json
export const resourceTypeOptions = [{
   label: i18n.t('Node'),
   value: 'node'
}, {
   label: i18n.t('Zone'),
   value: 'domain'
}, {
   label: i18n.t('AccountTemplate'),
   value: 'account_template'

... Continue here...
```
This is purely hypothetical - depending on what exactly needs optimization, other methods can be more appropriate or useful, but without further context, my recommendation stands.